tests/factories/reports/cht/pregnancy-visit.js

```diff
@@ -0,0 +1,291 @@
+const Factory = require('rosie').Factory;
+const uuid = require('uuid');
+
+const defaultFields = {
+  'inputs': {
+    'meta': {
+      'location': {
+        'lat': '',
+        'long': '',
+        'error': '',
+        'message': ''
+      },
+      'deprecatedID': ''
+    },
+    'source': 'contact',
+    'source_id': '',
+    'contact': {
+      '_id': '091d10d9-2f5c-4965-9c20-96730c6fc300',
+      'name': 'James',
+      'short_name': '',
+      'patient_id': '96854',
+      'date_of_birth': '1997-11-18',
+      'sex': 'female',
+      'parent': {
+        '_id': '8935e9d8-0b2a-5eb0-ae54-ad1377a60097',
+        'parent': {
+          'contact': {
+            'chw_name': '',
+            'phone': ''
+          }
+        }
+      }
+    }
+  },
+  'patient_age_in_years': '23',
```

Comment:
This value will be inaccurate over time. Please calculate it based on `date_of_birth`. Also, I suspect that `date_of_birth` is kind of arbitrary, but intended to be in some rough range. It is preferable that it be dynamic so it stays with a logical date for what the purpose is. The idea is probably to have "a 23 year old female" so then let's just calculate that.

Reply:
I think this is worth a discussion. We are going to find a bunch of values that are specific to the contact the report is filed for but this is based on a submitted report. Now that I think about it. It probably would make sense when building a report to pass in the data for whom it is about and then pull that off to fill these fields.  Would you be ok with a follow up PR after this one to define that?

Reply:
Yup, and you're right, there are probably a lot of cases where we care what the values are and they cross logical boundaries. I'm ok with a followup PR. No matter what, there is a lot of improvement here!